app/components/datafruits-chat-input-message.js

```diff
@@ -24,6 +24,14 @@ export default class DatafruitsChatInputMessage extends Component {
     this.set('showingGifSearch', false);
   }
 
+
+  @action
+  updateBigGifPreview(gif){
+
+    this.set('focusedGif', gif.url);
```

Comment:
Not sure if this would solve your problem of the lazy-image not re-rendering or not, but have you tried using tracked properties?
https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/

Even if it doesn't, you should give it a try as the API is generally a bit nicer, and you won't have to use `set` anymore.